frontend/src/app/commonComponents/MaintenanceBanner.tsx

```diff
@@ -4,12 +4,14 @@ import Alert from "./Alert";
 
 interface MaintenanceMode {
   active: boolean;
+  header: string;
   message: string;
 }
 
 export const MaintenanceBanner: React.FC = () => {
   const [maintenanceMode, setMaintenanceMode] = useState<MaintenanceMode>({
     active: false,
+    header: "SimpleReport alert.",
```

Comment:
Think this should be a `:`, not a period

Reply:
thanks, fixed

Reply:
The TF flow should default to `SimpleReport alert:` will test it in `test`. but will add a ticket to make the header changeable in there